README.md

```diff
@@ -381,6 +381,12 @@ Differences from `query()`:
     ```
     DATABASE_URL=mysql://localhost/my_database
     ```
+    
+    For a file-based database like SQLite, you must set absolute path using `$PWD` variable, where `$PWD` will be set to the location of the `.env` file.
+    
+    ```
+    DATABASE_URL=sqlite://$PWD/my_database.db
+    ```
```

Comment:
There's a lot wrong with this:

* `$PWD` will only be set if the current binary was invoked by a shell.
* `$PWD` isn't guaranteed to be relative to the `.env` file.
* You don't *need* to use an absolute path, relative paths are allowed:
```dotenv
# The SQLite driver interprets URLs differently than the other drivers, this is a relative path
DATABASE_URL=sqlite://my_database.db
```

But if you're in a workspace then the path will be relative to the _workspace_ root and not the root of the current crate.

Reply:
@abonander thanks!  I just double checked, it does indeed work like you describe.  I did run into an issue though - the doc tests keep complaining that the database is not set, while all other tests pass ok.  I have set the db location in the .env file in the root of my workspace Cargo.toml.  Do you know why it might be?  BTW, ideally, a resolved (absolute) database location should be shown in the error message. Thx!